cmd/kapacitord/main.go

```diff
@@ -89,15 +89,31 @@ func (m *Main) Run(args ...string) error {
 
 		signalCh := make(chan os.Signal, 1)
 		signal.Notify(signalCh, os.Interrupt, syscall.SIGTERM)
+		reloadSignal := make(chan os.Signal, 1)
+		signal.Notify(reloadSignal, os.Interrupt, syscall.SIGHUP)
```

Comment:
I think we'll still need the loop since the `SIGHUP` doesn't necessarily crash the runtime anymore.